File: ios/ProgressiveBlurViewManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(ProgressiveBlurViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(direction, NSString)
RCT_EXPORT_VIEW_PROPERTY(intensity, NSNumber)
RCT_EXPORT_VIEW_PROPERTY(tint, NSString)
RCT_EXPORT_VIEW_PROPERTY(locations, NSArray)

@end

File: ios/ProgressiveBlurViewManager.swift
import React

@objc(ProgressiveBlurViewManager)
class ProgressiveBlurViewManager: RCTViewManager {
  
  override func view() -> UIView! {
    return ProgressiveBlurView()
  }
  
  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
}
